From 12023b030b680cfcb465acd9da959d31dafed2f7 Mon Sep 17 00:00:00 2001
From: Dominick Grift <dac.override@gmail.com>
Date: Fri, 12 Apr 2019 20:15:35 +0200
Subject: [PATCH 03/29] nspawn: Fix volatile SELinux label

nspawn should associate the specified nspawn container apifs object label instead of the nspawn container process label with the volatile tmpfs

(cherry picked from commit 8f1ed04ad6646aeb1f2f28d485b17110c084de5b)
---
 src/nspawn/nspawn.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nspawn/nspawn.c b/src/nspawn/nspawn.c
index e05fa3dbf2..0f64aa68f3 100644
--- a/src/nspawn/nspawn.c
+++ b/src/nspawn/nspawn.c
@@ -3262,7 +3262,7 @@ static int outer_child(
                         arg_userns_mode != USER_NAMESPACE_NO,
                         arg_uid_shift,
                         arg_uid_range,
-                        arg_selinux_context);
+                        arg_selinux_apifs_context);
         if (r < 0)
                 return r;
 
-- 
2.21.0

